src/rebuild/README.md

```diff
@@ -1,275 +1,150 @@
-# Self-healing
+# Self-healing (Rebuild)
 
-**Topiccs in this Chapter:**
+In DAOS, if the data is replicated with multiple copies on different
+targets, once one of the target failed, the data on it will be rebuilt
+on the other targets automatically, so the data integrity will not be
```

Comment:
"data integrity" is opposite of data corruption, rebuild is for recovering "data redundancy" right?

Reply:
ah, yes